### PR TITLE
Doc changelog 20240131

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,52 @@
 Synapse Changelog
 *****************
 
+v2.161.0 - TBD
+==============
+
+Features and Enhancements
+-------------------------
+- Add item specific error message when users do not have sufficient permissions
+  on a object which is using easyperms.
```
Comment (Cisphyx) [inline CHANGELOG.rst:15]:
```suggestion
  on an object which is using easyperms.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,6 +6,52 @@
...
+  (`#3532 <https://github.com/vertexproject/synapse/pull/3532>`_)
+- Ensure that Nexus events which are written to the log are always applied and
+  cannot be cancelled while the Nexus handler is running.
+  (`#3518 <https://github.com/vertexproject/synapse/pull/3518>`_)
+- Add AHA information to the output of the ``Cell.getCellInfo()`` API. This
+  includes the the service name, leader, and network.
```
Comment (Cisphyx) [inline CHANGELOG.rst:21]:
```suggestion
  includes the service name, leader, and network.
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -6,6 +6,52 @@
...
+  (`#3519 <https://github.com/vertexproject/synapse/pull/3519>`_)
+- Logs related to AHA Service registration and setting services as offline are
```
Comment (Cisphyx) [inline CHANGELOG.rst:23]:
```suggestion
- Logs related to AHA service registration and setting services as offline are
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -6,6 +6,61 @@
...
+  includes the service name, leader, and network.
+  (`#3519 <https://github.com/vertexproject/synapse/pull/3519>`_)
+- Logs related to AHA service registration and setting services as offline are
+  now logged at the ``INFO`` level.
+  (`#3534 <https://github.com/vertexproject/synapse/pull/3534>`_)
+- When creating Cron jobs and Triggers, record their creation time.
+  (`#3521 <https://github.com/vertexproject/synapse/pull/3521>`_)
+  (`#3538 <https://github.com/vertexproject/synapse/pull/3538>`_)
+- Update the allowed versions of the ``cbor2`` and `` pycryptodome``
+  librariess
```
Comment (MichaelSquires) [inline CHANGELOG.rst:33]:
```suggestion
  libraries.
```

**Review by MichaelSquires — CHANGES_REQUESTED** [review]
One little typo :)

File: CHANGELOG.rst [hunk 5]
```diff
@@ -6,7 +6,7 @@
 Synapse Changelog
 *****************
 
-v2.161.0 - 2024-02-02
+v2.161.0 - 2024-02-06
```
Comment (Cisphyx) [inline CHANGELOG.rst:9]: Is this intentionally set to tomorrow?

Comment (vEpiphyte) [reply CHANGELOG.rst:9]: Yes I set it to today for burnin time :)

**Review by invisig0th — APPROVED** [review]

**Pull request merged**